tests/clients/cli/wrapper.go

```diff
@@ -522,3 +522,28 @@ func (w *Wrapper) WaitForPushLogByPeerEvent(id peer.ID) error {
 func (w *Wrapper) WaitForPushLogFromPeerEvent(id peer.ID) error {
 	return w.node.WaitForPushLogFromPeerEvent(id)
 }
+
+func (w *Wrapper) CreateDocIndex(
+	ctx context.Context,
+	col client.Collection,
+	doc *client.Document,
+) error {
+	panic("client side database")
```

Comment:
todo: Please implement these :)

Reply:
this is not supposed to be called from outside

Reply:
Then it shouldn't be in the `client` package

Reply:
Added error as discussed in syncup